language_service/src/state/tests.rs

```diff
@@ -605,20 +605,22 @@ async fn target_profile_update_causes_error_in_stdlib() {
     );
 }
 
-#[test]
-fn notebook_document_no_errors() {
+#[tokio::test]
```

Comment:
Are the tokio tests needed because these are async tests?

Reply:
Yup. The standard `#[test]` doesn't support async code, but `#[tokio::test]` does. I hadn't actually seen that before digging into the tests in this file, so it's new to me as well.